docs/how-to/configure-text-generation-models.md

```diff
@@ -0,0 +1,160 @@
+# Configure Text Generation Models
+
+KubeAI supports the following engines for text generation models (LLMs, VLMs, ..):
+
+- vLLM (Recommended for GPU)
+- Ollama (Recommended for CPU)
+- Need something else? Please file an issue on [GitHub](https://github.com/substratusai/kubeai).
+
+There are 2 ways to install a text generation model in KubeAI:
+- Use Helm with the `kubeai/models` chart.
+- Use `kubectl apply -f model.yaml` to install a Model CRD.
+
+KubeAI comes with pre-validated and optimized Model configurations for popular text generation models. These models are available in the `kubeai/models` Helm chart and
+are also published as raw manifests in the `manifests/model` directory.
+
+You can also easily define your own models using the Model CRD directly or by using the `kubeai/models` Helm chart.
+
+## Install a Text Generation Model using Helm
+
+KubeAI provides a `kubeai/models` chart that contains the pre-configured models.
+
+You can take a look at all the pre-configured models in the chart's default values file.
+
+```bash
+helm show values kubeai/models
+```
+
+### Install Text Generation Model using CPU
+
+Enable the `gemma2-2b-cpu` model using the Helm chart:
+
+```bash
+helm upgrade --install --reuse-values kubeai-models kubeai/models -f - <<EOF
+catalog:
+  gemma2-2b-cpu:
+    enabled: true
+    engine: OLlama
+    resourceProfile: cpu:2
+    minReplicas: 1 # by default this is 0
+EOF
+```
+
+### Install Text Generation Model using L4 GPU
+
+Enable the Llama 3.1 8B model using the Helm chart:
+
+```bash
+helm upgrade --install --reuse-values kubeai-models kubeai/models -f - <<EOF
+catalog:
+  llama-3.1-8b-instruct-fp8-l4:
+    enabled: true
+    engine: VLLM
+    resourceProfile: nvidia-gpu-l4:1
+    minReplicas: 1 # by default this is 0
+EOF
+```
+
+## Install a Text Generation Model using kubectl
+You can use the Model CRD directly to install a model using `kubectl apply -f model.yaml`.
+
+### Install Text Generation Model using CPU
```

Comment:
I dont think we need to duplicate the steps for CPU... I think we could just point to an example model after defining steps for install models above.

Reply:
As a user I prefer to have the full examples in the doc. We can link to more examples in the repo but I want to at least show CPU and GPU for both helm and non-helm flow.